### DISCUSSION TITLE
for accessing relational fields

### DISCUSSION THREAD

**Discussion by nitish7426** [pocketbase/pocketbase#2227] [Q&A] [answered]
I have two collections-
1. movies
2. genres
I added genres as relation in movies collection.

How can I access the movies related to specific genre?

**Comment by ganigeorgiev** [comment]
From the description I'm not sure that I understand where the `relation` field is located.

In any case, you can send 2 separate requests OR check [Expanding relations](https://pocketbase.io/docs/expanding-relations/).

**Reply by ganigeorgiev** [reply, marked as answer]
On second read, if you want to filter `movies` based on a specific `genres` field, you can use something like (_assuming that `genres` is a multiple `relation` field in `movies`_):

```js
const genre = "action";
await pb.collection("movies").getList(1, 30, { filter: `genres.name ?= "${action}"` });
```

**Reply by nitish7426** [reply]
It is showing invalid filtered parameter

**Reply by ganigeorgiev** [reply]
@nitish7426 The above code was just an example (eg. I don't know if your `genres` collection has a `name` text field). The general idea is to use dot-notation if you want to access the relation fields.

If that doesn't help, please at least provide a code sample what you are doing and a little more details about your collections structure (you can export it as json from "Admin UI > Settings > Export collections").

**Reply by nitish7426** [reply]
Thank you for your help.

**Comment by nitish7426** [comment]


**Reply by ganigeorgiev** [reply]
@nitish7426 The screenshot doesn't show what are the fields in your genres collection. Please check the above reply - https://github.com/pocketbase/pocketbase/discussions/2227#discussioncomment-5544224.
